tests/playbooks/tests_auto_dns.yml

```diff
@@ -0,0 +1,80 @@
+# SPDX-License-Identifier: BSD-3-Clause
+---
+- name: Test output device of routes
+  hosts: all
+  vars:
+    type: veth
+    interface: ethtest0
+
+  tasks:
+    - name: Include the task 'show_interfaces.yml'
+      include_tasks: tasks/show_interfaces.yml
+    - name: Include the task 'manage_test_interface.yml'
+      include_tasks: tasks/manage_test_interface.yml
+      vars:
+        state: present
+    - name: Include the task 'assert_device_present.yml'
+      include_tasks: tasks/assert_device_present.yml
+
+    - name: Test the route or the warning log when configuring the route with
+        or without the interface name
+      block:
+        - name: Configure an interface with auto_dns enabled
```

Comment:
the test uses `auto_dns: false` - shoud the name be `Configure an interface with auto_dns disabled`?